ccm_web/client/src/models/feature.ts

```diff
@@ -43,7 +43,7 @@ const formatThirdPartyGradebookProps: FeatureDataProps = {
 const createSectionsProps: FeatureDataProps = {
   id: 'CreateSections',
   title: 'Create Sections',
-  description: 'Create sections through csv files into your own course',
+  description: 'Create sections through a CSV file in your course',
```

Comment:
@janelilr, I made more changes here than were discussed. Upon closer inspection, I found the language confusing, so I changed to a singular file, changed the preposition to "in", and removed "own", which seemed unnecessary. Let me know what you think.